doc/md/getting-started.md

```diff
@@ -141,12 +141,12 @@ import (
 func main() {
```

Comment:
The change is not needed in this file, because we format and fail.

Reply:
👍 will fix this and update for changes on `master`